src/static/js/index.js

```diff
@@ -40,6 +40,13 @@ if (!Array.prototype.indexOf)
 */
 
 documentReady( function() {
+  buildCharts();
+
+  window.addEventListener( 'resize', buildCharts );
```

Comment:
So the actual feature we need to add here is called debounce. See this helpful description:

https://forum.highcharts.com/highcharts-usage/performance-recommendation-for-window-resize-reflow-t14452/

>  it makes more sense to "debounce" the event so the reflow function itself doesn't even fire and run through its logic until the window has stopped resizing.

More info: [Throttle vs debounce](http://benalman.com/projects/jquery-throttle-debounce-plugin/)

Adding a debounce handler will improve performance of this PR's code when the user resizes a window or changes their mobile device's orientation.

However, I would still recommend we try to make these styles responsive with CSS instead of using JS and the responsive options; CSS is a simpler solution that will be much more performant for the end user. Only IF we cannot accomplish the design with CSS, should we attempt to make the JS do this. I'm happy to take a look at CSS / highcharts styling docs and see if I can prototype something that we can improve upon for matching AJ's designs.